<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="settings_tabs">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="settings_appearance_tab">
         <attribute name="title">
          <string>Appearance</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QWidget" name="widget_10" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="DarkThemeBtn">
                  <property name="text">
                   <string>Dark Theme</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="LightThemeBtn">
                  <property name="text">
                   <string>Light Theme</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="settings_recordings_tab">
         <attribute name="title">
          <string>Recordings</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QWidget" name="widget_7" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QWidget" name="widget_9" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Recording file format</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="saveFormatComboBox"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_8" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Save records to</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="saveRootTextEdit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="SelectDataDirBtn">
                  <property name="text">
                   <string>Browse</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="settings_video_device_tab">
         <attribute name="title">
          <string>Video Device</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QWidget" name="widget_11" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="reload_video_device_button">
               <property name="text">
                <string>Reload Video Devices</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="settings_audio_device_tab">
         <attribute name="title">
          <string>Audio Device</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QPushButton" name="reload_audio_device_button">
            <property name="text">
             <string>Reload Audio Devices</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="settings_streams_tab">
         <attribute name="title">
          <string>Streams</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Line chart visualization mode</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="linechart_viz_mode_combobox"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Plot frame per second</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="plot_fps_lineedit"/>
                </item>
               </layout>
              </widget>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="QPushButton" name="reload_stream_preset_button">
               <property name="maximumSize">
                <size>
                 <width>256</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Reload stream presets</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="general_tab">
         <attribute name="title">
          <string>General</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="reset_default_button">
            <property name="text">
             <string>Reset to default</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
